conda-forge.yml

```diff
@@ -21,4 +21,12 @@ github:
 provider:
   linux_aarch64: default
   linux_ppc64le: default
+  linux_64: github_actions
+  win_64: github_actions
```

Comment:
You don't have the right configuration for using windows runners. For that you need to do something like https://github.com/conda-forge/.cirun/pull/108 (and get permission from the prefix folks).